apollo-sample/src/main/java/com/example/apollographql/sample/feed/GitHuntFeedActivity.java

```diff
@@ -0,0 +1,82 @@
+package com.example.apollographql.sample.feed;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.util.Log;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ProgressBar;
+
+import com.apollographql.apollo.ApolloCall;
+import com.apollographql.apollo.api.Response;
+import com.apollographql.apollo.cache.normalized.CacheControl;
+import com.apollographql.apollo.exception.ApolloException;
+import com.example.FeedQuery;
+import com.example.apollographql.sample.R;
+import com.example.apollographql.sample.GitHuntApplication;
+import com.example.apollographql.sample.detail.GitHuntEntryDetailActivity;
+import com.example.type.FeedType;
+
+import javax.annotation.Nonnull;
+
+public class GitHuntFeedActivity extends AppCompatActivity implements GitHuntNavigator {
+
+  private static final String TAG = GitHuntFeedActivity.class.getSimpleName();
+
+  private GitHuntApplication application;
+
+  private ViewGroup content;
+  private ProgressBar progressBar;
+  private GitHuntFeedRecyclerViewAdapter feedAdapter;
```

Comment:
super nitpick but private fields accessed from inner classes do a +1 on method count
https://medium.com/@dineshbob10/effects-of-javas-synthetic-accessor-methods-in-android-bb67b3bac22e